embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/session/orchestrator/SessionSpanAttrPopulatorImpl.kt

```diff
@@ -54,20 +51,10 @@ internal class SessionSpanAttrPopulatorImpl(
             endType?.toString()?.lowercase(Locale.US)?.let {
                 addSystemAttribute(SpanAttributeData(embSessionEndType.name, it))
             }
-
-            val startupInfo = when {
-                coldStart -> eventService.getStartupMomentInfo()
-                else -> null
-            }
-            startupInfo?.let { info ->
-                addSystemAttribute(
-                    SpanAttributeData(
-                        embSdkStartupDuration.name,
-                        sdkStartupDurationProvider(coldStart).toString()
-                    )
-                )
-                addSystemAttribute(SpanAttributeData(embSessionStartupDuration.name, info.duration.toString()))
-                addSystemAttribute(SpanAttributeData(embSessionStartupThreshold.name, info.threshold.toString()))
+            if (coldStart) {
+                startupService.getSdkStartupDuration()?.let { duration ->
```

Comment:
This is fine as long as the service is populated before then, when `StartupService.setSdkStartupInfo()` is called during init.

I will say though - is this actually used? I can confirm with the backend folks, but the `emb-sdk-startup` span records the same information but provides much more context. 

If it's init ordering is OK, this should be OK. But if we can rip it out, then it's probably best to do that. I'll confirm.